src/harness/harness.ts

```diff
@@ -732,7 +732,8 @@ module Harness {
     }
 
     // Settings 
-    export var userSpecifiedroot = "";
+    export let userSpecifiedRoot = "";
+    export let lightMode = false;

```

Comment:
We probably don't want this as a global.  Instead, decide this on a test by test basis.